src/components/Resume/Resume.js

```diff
@@ -2,6 +2,7 @@ import React, { forwardRef, useEffect, useRef, useState } from "react";
 import { Calendar } from "react-feather";
 import joshImage from "../../assets/Josh-Logo-White-bg.svg";
 import { genderOptions, getMonthString } from "../../Constants";
+import jsonData from "./jsonData.json";
```

Comment:
remove unnecessary code. this PR should only have navbar and layout related changes.

Reply:
done